File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/PostPreviewPage.java
package org.wordpress.android.e2e.pages;

import androidx.test.espresso.ViewInteraction;

import org.wordpress.android.R;

import static androidx.test.espresso.Espresso.onView;
import static androidx.test.espresso.assertion.ViewAssertions.matches;
import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
import static androidx.test.espresso.matcher.ViewMatchers.withId;

public class PostPreviewPage {
    private ViewInteraction mPreview;

    public PostPreviewPage() {
        mPreview = onView(withId(R.id.preview_container));
        mPreview.check(matches(isDisplayed()));
    }
}

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/PostPreviewPage.kt
package org.wordpress.android.e2e.pages

import androidx.test.espresso.Espresso
import androidx.test.espresso.ViewInteraction
import androidx.test.espresso.assertion.ViewAssertions
import androidx.test.espresso.matcher.ViewMatchers
import org.wordpress.android.R

class PostPreviewPage {
    private val mPreview: ViewInteraction

    init {
        mPreview = Espresso.onView(ViewMatchers.withId(R.id.preview_container))
        mPreview.check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
    }
}
